src/modules/obj.js

```diff
@@ -1,7 +1,9 @@
-export default class Obj {
-  constructor(NewValue, index) {
-    this.inputValue = NewValue;
-    this.indexNumber = index;
-    this.completed = true;
+class Obj {
+  constructor(value) {
+    this.inputValue = value;
+    this.index = 0;
```

Comment:
- Hey, the index value should be the length of the array and not zero, i.e if a 5th task is added to the list, the index of that task should equal to 5.